### PR TITLE
Use DOCtor-RST 1.60.1

### PR DESCRIPTION
**Pull request by OskarStark**
cc @alamirault @smnandre

### CODE REVIEW

**Comment by OskarStark** [comment]
New image not yet available...

**Comment by OskarStark** [comment]
All files are valid 🤔 , but this is 5.4, so this seems to be correct, right?

**Comment by alamirault** [comment]
@OskarStark doctor-rst file must be update 😂

**Comment by OskarStark** [comment]
Indeed, it's too late 😄

**Review by xabbuh — APPROVED** [review]

**Comment by OskarStark** [comment]
@smnandre can you please check the occurrences? One is a false positive, thanks

**Comment by OskarStark** [comment]
> In AbstractUnicodeString.php line 235:
>                                                         
>  Matching failed with error: Backtrack limit exhausted  

Any clue @smnandre ?

**Comment by smnandre** [comment]
> > In AbstractUnicodeString.php line 235:
> > Matching failed with error: Backtrack limit exhausted
> 
> Any clue @smnandre ?

Hmm maybe the positive lookahead in the pattern ... but it should only be called on a (short) php code line, no ?

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -73,7 +73,7 @@ jobs:
                     key: ${{ runner.os }}-doctor-rst-${{ steps.extract_base_branch.outputs.branch }}
 
             -   name: "Run DOCtor-RST"
-                uses: docker://oskarstark/doctor-rst:1.59.0
+                uses: docker://oskarstark/doctor-rst:1.60.0
```
Comment (OskarStark) [inline .github/workflows/ci.yaml]:
```suggestion
                uses: docker://oskarstark/doctor-rst:1.60.1
```

**Pull request merged**